t/ssl.t

```diff
@@ -2065,3 +2065,106 @@ qr/\[error\] .*? failed to parse pem key: PEM_read_bio_PrivateKey\(\) failed/
 [alert]
 [emerg]
 [crit]
+
+
+
+=== TEST 20: read client addr via ssl.raw_client_addr()
+--- http_config
+    lua_package_path "$TEST_NGINX_LUA_PACKAGE_PATH/?.lua;;";
+
+    server {
+        listen 127.0.0.1:12345 ssl;
+        server_name   test.com;
+        ssl_certificate_by_lua_block {
+            local ssl = require "ngx.ssl"
+            local byte = string.byte
+            local addr, addrtype, err = ssl.raw_client_addr()
+            local ip = string.format("%d.%d.%d.%d", byte(addr, 1), byte(addr, 2),
```

Comment:
@smartwjw 
Better to localize `string.format` just like the `string.byte`?

Reply:
For test cases, I think it's fine, but for documentation, it's worth it since the user usually just does copy&paste :)